core-web/libs/portlets/edit-ema/portlet/src/lib/store/features/editor/withEditor.ts

```diff
@@ -129,7 +129,11 @@ export function withEditor() {
                     const showBlockEditorSidebar = canEditPage && isEditState && isEnterprise;
 
                     const canUserHaveContentletTools =
-                        !!contentletArea && canEditPage && isEditState && !isScrolling;
+                        !!contentletArea &&
+                        canEditPage &&
+                        isEditState &&
+                        !isScrolling &&
+                        !isPreview;
```

Comment:
A lot of conditional without control. Let's take note to fix this in the UVE & SDK Architecture Improvements

Reply:
I think we should have 2 states and 2 components for edit and preview, this is just getting wayyyy to crazy.

Reply:
We even have that as an enum in the sdk.